BadHabbit/BadHabbit/SceneDelegate.swift

```diff
@@ -22,10 +21,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         }
         
         let window = UIWindow(windowScene: windowScene)
-        
+
         self.appAssembly = AppAssembly(application: .shared, window: window)
-        window.rootViewController = appAssembly.storiesAssembly.signUpScreen.assembleStory()
-        
+
+        let navigationController = UINavigationController(rootViewController: appAssembly.storiesAssembly.signUpScreen.assembleStory())
```

Comment:
embedding in Navigation in view assembly, not in scene

Reply:
changed